gallery/gallery/lib/pages/about.dart

```diff
@@ -20,6 +21,11 @@ void showAboutDialog({
   );
 }
 
+Future<String> getVersionNumber() async {
+  PackageInfo packageInfo = await PackageInfo.fromPlatform();
```

Comment:
Can you check if this works on web and macOS? If it doesn't thats ok but maybe add a comment about that

Reply:
The package doesn't support web and macOS yet. I'll see if I can add it for macOS, otherwise will create an issue